### ISSUE TITLE
Joyread initial release progress

### ISSUE THREAD

**Issue by mysticmode** [joyread/server#15]
Hi everyone!

I've seen few people who had shown interest in this project and in getting updates. So, I'm creating this issue just for sharing the progress. Technical issues might go in separate Github issues or in Discord
https://discord.gg/TQSngr

Just wanted to say, I've been juggling with quiet a few ideas. I was thinking to make this project commercial, like https://www.pancakeapp.com/ (and that is the reason I was moving the repo here and there).

But that commercial idea is not gonna work for me personally. So, honestly this project will be on Github and be AGPLv3 licensed software forever. I might build a SaaS around this project in the future, but that is another story.

Also, I got a job recently, so things might be slow - but I will continue working on this.

So, try to keep in touch!
mysticmode

cc @sunjam @fivestones @ocdtrekkie @AntonioBusillo @GAumala

**Comment by mysticmode** [comment]
I've worked on the PDF viewer page. This is going to be a first of many iterations to come. I'm just sharing the screenshot

![screencapture-localhost-8080-viewer-nextcloudmanual-pdf-2018-12-16-00_12_16](https://user-images.githubusercontent.com/1125199/50046299-7c92d400-00c7-11e9-978f-d9442f589524.png)

Uploading PDF files on Joyread is ready for now.

**Comment by mysticmode** [comment]
Currently, I'm only focusing on the reader view for PDF, EPUB, CBZ & CBR including Highlights & Annotations. This is a very crucial part in the app, if we don't get this working well, responsive and polished, I think not many people would use this.

So, once this is ready, then I'll move on to implementing multi-user functionality in the app and other features.

**Comment by mysticmode** [comment]
Just FYI: Previously LibreRead had collections where you can collate books into a collection. But I think adding tags to books (while you are uploading or later) is more efficient than collections. You can search for tags and find books. Or you can select it in the home page. Screenshot shown below

![tags](https://user-images.githubusercontent.com/1125199/50095071-a0c2f200-023a-11e9-956c-e42eaef7b506.PNG)
